en/searcher-development.html

```diff
@@ -642,6 +642,29 @@ <h2 id="error-handling">Error handling</h2>
 </p>
 
 
+<h2 id="timeouts">Timeouts</h2>
+<p>
+  How to gracefully handle a timeout inside a Searcher?
+  <code>Result result = execution.search(query)</code> can result in a timeout, and is not null.
```

Comment:
What does this mean - "and is not null" ?

Reply:
search() returns a non-null object - but can remove this, not important.